x-pack/plugins/global_search_bar/server/config.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { PluginConfigDescriptor } from '@kbn/core/server';
+import { schema, TypeOf } from '@kbn/config-schema';
+
+export const configSchema = schema.object({
+  input_max_limit: schema.number({
```

Comment:
The new configuration setting should be documented here: https://www.elastic.co/guide/en/kibana/current/settings.html

Reply:
As agreed in slack, it's fine to have this configuration setting undocumented.